docker/php-8.2/xdebug.ini

```diff
@@ -0,0 +1,14 @@
+;zend_extension = xdebug.so
+
+; XDebug 3 → https://xdebug.org/docs/upgrade_guide
+; You can dynamically enable XDebug by setting XDEBUG_MODE env variable.
+; Some options can be dynamically overridden with XDEBUG_CONFIG env variable.
+xdebug.mode = off
+xdebug.start_with_request = yes
+xdebug.discover_client_host = true
+xdebug.client_host = host.docker.internal
+xdebug.client_port = 9003
+
+; Required so XDebug DOES NOT print warning "Could not connect to debugging client"
+xdebug.log = /app/docker/php-8.2/xdebug.log
+xdebug.log_level = 1
```

Comment:
what is wrong with "could not connect to debugging client" warning?

Reply:
It's annoying when you use CLI and output is flooded with this warning every time.